### DISCUSSION TITLE
2022-06-08 Town Hall #4

### DISCUSSION THREAD

**Discussion by jim-stanczak** [cxndao/constellation#21] [Meeting Notes]
### Marketing & PR

- Our Twitter Account ('@CXNDAO) is slowly getting off the ground, but we could use help!
- Be on the lookout for more information on potential Bounties
- @ dan green has a free MailChimp account set up, so launch is imminent
    - Looking to simplify the validation process for new members
    - Create scripts for inbound info gathering.
- Brian is bullish on the initiative, given his experience at CreatorLand
- Meeting to be set on vision and mission statement for any who can attend and participate, voting and feedback to be gathered after 

### Website
- Voting on a Domain Name!
- Need to settle on a content strategy (Should this be a subcommittee?)

### Fund Updates
- Meetings are still indicating demand for Web3 funds from investors
- We need to seed the fund with a significant investment from the DAO to validate it is an alumni-driven effort
    - Discussion around dollar-amount or % of DAO participation as a key KPI
- Future discussions on developing investment criteria
    - Seeking 1 company per cohort
    - Web3 based business
    - Instruments: SAFT, MFN, Equity & Tokens
    - 20% allocation to non-thesis investments

### NFT Drop
- Coming soon!
- Looking to leverage the drop + Fund PR into a significant bump in community engagement

### Governance
- MJ giving a shoutout to @ Matt Gallagher for making great progress on CIP-1. Check the discord Thread with links here: <https://discordapp.com/channels/920456009267429417/920456010014011444/984127108244721664>

**Comment by mattjoyal** [comment]
Thanks @jim-stanczak for adding this. 

## Meeting Notes

### Planned Agenda

- Membership NFT Status
- Social Media - Twitter
- What is a Bounty?
    - Example
- Website Status
- Email Distribution List - *Brian*
- Fund Status - *Tak*
- Press Release - *Tak*
- CIP-1 - Proposals Process
- Web3 State of Health
- looking for ways to send announcements when people are not frequently checking Discord
    - connect with those more used to web2 traditional messaging

## Pre-Amble

- MailChimp Subscription will be great
- Zeugh - Origami representative

## Web3 State of Health

- Notion - tracking & documenting to curate list of links, investor links
- DAO impact - helping other TS companies raise, help hiring

## Membership NFT Status

- visual NFT progress on design
    - static version
- meeting this afternoon to go over design, finalize
    - mint date (window) to be solidified

## Social Media - Twitter

- Jim - started up CXN Twitter
    - started the basics, open to it on vision & who else wants to work on it
- PR event - pin that tweet
- 2 pain points, hiring & fundraising
    - good topics for content → tweet & retweet around these topics, education

## What is a Bounty?

- bounty - way to get compensated from a DAO
    - get tokens for completing the bounty
- Assigned, discrete task - good for intro into DAOs
- [[Dework.xyz](http://dework.xyz/)](http://Dework.xyz) - platform for bounties
    - How to access? See with Todd if we have one already

## Website Status

- General - request on vote for CXN website names

## Email Distribution List - *Brian*

- start automating these things
    - auto-follow on twitter
    - free tier, should work for us for a while
- example - joinjump.community
    - clean & simple implementation

## Fund Status - *Tak*

- Angellist call - structure, accounts set up, aiming at end of next week
- $10M fundraise
- Kickstart this:
    - DAO members first, & family+friends
        - alumni driven if DAO members invest themselves
        - 20% token pool - early committers & DAO participation
            - pro rated extra tokens
- Deck, talk about web3 & alumni
    - TS companies & returns
    - Tak needs any versions of Techstars fund decks
        - start with Cody
- percent participation - CXN members, check size low - optimize for contribution rate
- story: alumni able to help & fund other alumni
- types of companies to invest in: criteria
    - 1 company per cohort & web3 based
    - take equity & tokens, flexibility here to invest in both
    - as investing, adjust accordingly - not expecting issue in lack of companies
        - non-thesis opportunities available too

## Press Release - *Tak*

- sending link mentioned
- Also send an op-ed out
- need to get website live
- Dan - TechCrunch contact
- committee team meetings - how make sure our members see this?
    - discussion to continue offline
    - NFT drop, ideally draw more attention, do first

## CIP-1 - Proposals Process

- Shout to MG for the hard work
- get this merged in the coming weeks, get to a vote

- GitBook - central repository
    - we need a spot to centralize all services we use
    - commercial solution for newer users
    
- Vision statement
    - Tak / Jim to review & take a swing at the copy
    - review in focus call
    - 
- Founder spotlight idea
    - highlight founder in CSN and what they are working on
